core/src/ast_builder/expr/function.rs

```diff
@@ -349,12 +369,26 @@ pub fn radians<V: Into<ExprNode>>(expr: V) -> ExprNode {
     ExprNode::Function(Box::new(FunctionNode::Radians(expr.into())))
 }
 
+pub fn ltrim<T: Into<ExprNode>>(expr: T, chars: Option<T>) -> ExprNode {
+    ExprNode::Function(Box::new(FunctionNode::Ltrim(
+        expr.into(),
+        chars.map(|chars_node| chars_node.try_into().unwrap()),
+    )))
+}
+
+pub fn rtrim<T: Into<ExprNode>>(expr: T, chars: Option<T>) -> ExprNode {
+    ExprNode::Function(Box::new(FunctionNode::Rtrim(
+        expr.into(),
+        chars.map(|chars_node| chars_node.try_into().unwrap()),
+    )))
+}
+
```

Comment:
You have to change it here too.

Reply:
Oh, thank you for pointing this out, i will make those changes ASAP